### PR TITLE
MoEngage Actions Updated Doc

### PR DESCRIPTION
**Pull request by aekanshd**
### Proposed changes

1. Updated the doc by removing the mapping section as the table was empty. (Suggested in https://github.com/segmentio/segment-docs/pull/3395#issuecomment-1224286537)
2. Removed unnecessary sections.

### Merge timing
ASAP once approved

### CODE REVIEW

File: src/connections/destinations/catalog/actions-moengage/index.md [hunk 1]
```diff
@@ -63,14 +63,10 @@ Additional Context
...
-### Mapping the User ID and Anonymous ID
+### Default Mapping
 
-To map the User ID to the Anonymous ID, make sure you the correct values. Scroll down to the **Define mappings** section and change the following keys from sentence case to Camel Case. Make sure you select the new values from the dropdown.
-
-Key | Default Value | New Value
----------|----------|---------
- User ID | `user_id` | `userId`
- Anonymous ID | `anonymous_id` | `anonymousId`
+> warning "Do not change the preset values"
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-moengage/index.md:68]:
```suggestion
> warning ""
> Don't change the preset values
```

File: src/connections/destinations/catalog/actions-moengage/index.md [hunk 2]
```diff
@@ -63,14 +63,10 @@ Additional Context
...
+> We already provide preset values to all mappings and hence no changes are required in the Identify User Actions, and Track Event Actions.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-moengage/index.md:69]: @aekanshd who provides the preset values? Segment or MoEngage?

**Review by stayseesong — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/actions-moengage/index.md (hunk 2 above)
Comment (aekanshd) [reply src/connections/destinations/catalog/actions-moengage/index.md:69]: MoEngage provides the present values.

Comment (stayseesong) [inline src/connections/destinations/catalog/actions-moengage/index.md:69]:
```suggestion
> MoEngage provides preset values to all mappings. You don't need to make any changes to the Identify User Actions and Track Event Actions.
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

**Comment by stayseesong** [comment]
closing this PR as a separate PR (#3406) was created to edit the file

**Pull request closed without merging**